XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -170,9 +171,18 @@ A **`CFTokenIssuance`** can be removed using the same approach, but only if the
 Each **`CFTokenIssuance`** costs an incremental reserve to the owner account. This specification allows up to 32 **`CFTokenIssuance`** entries per account.
 
 #### 1.2.1.2. The **`CFToken`** object
+
 The **`CFToken`** object represents an amount of a token held by an account that is **not** the token issuer. CFTs are acquired via ordinary Payment or DEX transactions, and can optionally be redeemed or exchanged using these same types of transactions. The object key of the `CFToken` is derived from hashing the space key, holder's address and the `CFTokenIssuanceID`.
 
-##### 1.2.1.2.1 Fields
+##### 1.2.1.2.1. **`CFToken`** Ledger Identifier
+
+The ID of a CFToken object, a.k.a `CFTokenID` is the result of SHA512-Half of the following values, concatenated in order:
+
+* The CFTokenIssuance space key (0x007E).
+* The AccountID of the token holder.
+* The `CFTokenIssuanceID` for the issuance being held.
```

Comment:
```suggestion
* The `CFTokenIssuanceID` for the issuance being held.
* The AccountID of the token holder.
```

The order is actually reversed in my implementation. Sorry didn't account for the order in my previous spec PR